### DISCUSSION TITLE
How to add external device (usb drive) when admin disabled it.

### DISCUSSION THREAD

**Discussion by 1-Mythical** [3kh0/ext-remover#493] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Simply, my admin has disabled the ability to use any external devices. Downgrading does still work, but my files won't load when I have it launched and logged in.
![Screenshot 2023-02-06 12 00 35](https://user-images.githubusercontent.com/79861302/217036064-d7283b9e-2ff2-4c1b-aafe-ec8287ad5865.png)

**Comment by 3kh0** [comment, marked as answer]
Use a home computer, and when you use shimmer, like when you execute it, it should be on the recovery screen

**Reply by 1-Mythical** [reply]
SH1immer seems like it is too much. I don't want to get in trouble from that.  Many people told me schools have a way to notify when a Chromebook is unenrolled.

**Reply by 3kh0** [reply]
@MythicalTrashcan not really, it is just that if your chromebook says offline for quite some time, they can see it, so just reenroll every once in a while

**Reply by 1-Mythical** [reply]
How often should I re-enroll then. I don't want to do it every 10 minutes or too long that I get caught.

**Reply by obkduishinijf** [reply]
> How often should I re-enroll then. I don't want to do it every 10 minutes or too long that I get caught.

do it every few days

**Reply by xHydra1337** [reply]
Ive been unenrolled for ~a month lmao, kids break chromebooks here all the time it really doesnt raise a brow if a chromebook has been offline for a while. When you do however re enroll, dont sign in because they can see who was signed on last before it was un enrolled, that just raises suspicion.

**Reply by funkymokey** [reply]
so then wth do you re enroll for

**Reply by Holding-THE-CREATOR** [reply]
at the end of the year so that you don't get caught

**Reply by Holding-THE-CREATOR** [reply]
My admin uses GoGuardian and if a Chromebook is offline you are immediately
told to have it inspected. They make sure your Chromebook is enrolled after
that.
The solution is to downgrade but still be enrolled and use ingot to disable
lightspeed. Then you just don't use it in class unless you have a proxy
that doesn't show up to GoGuardian
